android/app/src/main/java/org/electroncash/electroncash3/Main.kt

```diff
@@ -493,6 +542,53 @@ class ChangePasswordDialog : AlertDialogFragment() {
 }
 
 
+class WalletExportDialog : AlertDialogFragment() {
+    override fun onBuildDialog(builder: AlertDialog.Builder) {
+        builder.setTitle(R.string.export_wallet)
+                .setView(R.layout.wallet_export)
+                .setPositiveButton(android.R.string.ok, null)
+                .setNegativeButton(android.R.string.cancel, null)
+    }
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): AlertDialog {
+        val dialog = super.onCreateDialog(savedInstanceState)
+        dialog.window!!.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_VISIBLE)
+        return dialog
+    }
+
+    @SuppressLint("SetTextI18n")
+    override fun onFirstShowDialog() {
+        val walletName = arguments!!.getString("walletName")!!
+        val extension = ".ecw"
```

Comment:
I notice that neither of the desktop wallet nor the iOS wallet give wallet save/export files an extension.  I think it's worth adding one unless this has already been explored...

Reply:
The most likely thing for the user to do with the file is to copy it to the desktop version, and then they'd have some wallets with extensions and some without, depending on which platform they were created on. So I think we'd better not add an extension unless we coordinate that change with the desktop version.

Reply:
Ok, I've taken out the extension from here for the time being and raised the issues here instead #1728.